unit_tests/helpers/analytics.py

```diff
@@ -0,0 +1,18 @@
+from app.model.analytics import SummaryDTO
+
+# ANALYTICS SUMMARY
+EXPECTED_NUM_DOCUMENTS = 33
+EXPECTED_NUM_FAMILIES = 22
+EXPECTED_NUM_COLLECTIONS = 11
+EXPECTED_NUM_EVENTS = 0
+
+
+def create_summary_dto(
```

Comment:
Probably don't need this - but its nice for symmetry :+1:

Reply:
I thought the same! Depends how we extend the analytics service going forwards I suppose.